week03_model_free/display_server_preparation.sh

```diff
@@ -0,0 +1,47 @@
+echo "apt-get updating..."
+apt-get -qq update > /dev/null
+echo "x11vnc installing if it is not exists..."
+echo ""
+apt-get -q install -y x11vnc
```

Comment:
Still more stuff is installed. I am thoroughly confused about what I need to install to run the environment with and without visualization.

Reply:
In my mind, for real visualization (not just streaming into some video buffer that cannot be seen) the script installs needed packages itself